dotnet/src/IntegrationTests/Planning/SequentialPlanner/SequentialPlanParserTests.cs

```diff
@@ -2,8 +2,8 @@
 
 using Microsoft.Extensions.Configuration;
 using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Planners.Sequential;
 using Microsoft.SemanticKernel.Planning;
-using Microsoft.SemanticKernel.Planning.Sequential;
 using SemanticKernel.IntegrationTests.Fakes;
 using SemanticKernel.IntegrationTests.TestSettings;
 using Xunit;
```

Comment:
Can you also update the namespaces and folder structures of the integration tests

We currently have `namespace SemanticKernel.IntegrationTests.Planning.SequentialPlanner;`

Change this to `namespace SemanticKernel.IntegrationTests.Planners.Sequential;` and folders to match

Reply:
Done